lib/auth/webauthn/httpserver/main.go

```diff
@@ -84,8 +86,15 @@ func run() error {
 	http.HandleFunc("/register/1", s.register1)
 	http.HandleFunc("/register/2", s.register2)
 
+	srv := &http.Server{
+		Addr:              *addr,
+		ReadTimeout:       apidefaults.DefaultIOTimeout,
+		ReadHeaderTimeout: defaults.ReadHeadersTimeout,
+		WriteTimeout:      apidefaults.DefaultIOTimeout,
+		IdleTimeout:       apidefaults.DefaultIdleTimeout,
+	}
 	fmt.Printf("Listening at %v\n", *addr)
-	return http.ListenAndServeTLS(*addr, *certFile, *keyFile, nil /* handler */)
```

Comment:
I would revert this file, like Zac said it doesn't matter and it'll only shift blame to you.

(I can probably delete this by now.)

Reply:
I would prefer to keep the change for consistency within the project.  It will help if we decide we want to adopt `gosec` scanning

Reply:
#30314